### ISSUE TITLE
KA Lite Monitor notifications

### ISSUE THREAD

**Issue by radinamatic** [learningequality/installers#370]
## Summary

Since VMs can be slow I don't always follow all the notifications with expectations, but go and multitask while they are working... :wink: 

Anyhow, I did notice that the messages `KA Lite stopped` and `You can now open the browser...` sometimes appear when the user is not expecting them. For example:

1. Server is stopped - user opens the menu and chooses `Start KA Lite`
   - first message appears: `Starting...`
   - then `Stopped` appears, but the icon is representing the running state and
   - no `You can now open the browser...` message 

2. Server is running  - user opens the menu and chooses `Stop KA Lite`
   - first message appears: `Stopping...`
   - then `You can now open the browser...`
   - finally the `Stopped` appears

## Branch or installer

0.16 (build 66) on (I think) all versions of OSX. 

## Screenshots

This would require captured video, do you have any Mac app suggestion for that  @cpauya?

**Comment by cpauya** [comment]
@radinamatic We use [Licecap](http://www.cockos.com/licecap/) to record a screen activity into a .gif - you have to experiment with your setup which ever works and efficient for you.

Those notifications have been "unreliable" - sometimes some doesn't even show when they're supposed to.  

I will check them out tomorrow.

**Comment by radinamatic** [comment]
@cpauya Licecap looks like just the right thing, thanks!  :+1:

Did you deactivate notifications completely in build 69? I didn't see any this time...

**Comment by radinamatic** [comment]
Actually, notification did not appear at all at first run, but did after I installed Spanish contentpack... :confused:

**Comment by cpauya** [comment]
Nope @radinamatic - I just addressed the issue you reported in this issue about the alert with wrong notifications.  

The notifications appearing/not appearing is the "unreliable" scenario that I mentioned earlier.  It sometimes doesn't show the alerts from the app but are recorded at the Notifications Center of OS X.

What I do is launch the Notification Center of OS X - and I can see all of the KA Lite notifications there.  After I do this, the alerts are now shown by OS X.  But expect to not see them again when you restart the VM.

Weird and it's actually on my ToDo list.

**Comment by radinamatic** [comment]
Thanks, @cpauya, I'll grab the build 71 once it's ready!

**Comment by cpauya** [comment]
Update: I have addressed the "unreliable" alert notifications on the https://github.com/learningequality/installers/pull/359#issuecomment-191195606 PR - that should fix this issue too when merged.

**Comment by radinamatic** [comment]
Ok @cpauya, is there anything specific to test on build 71, or I better wait for this to be merged?

**Comment by cpauya** [comment]
Nope, the fixes are already on build 71 @radinamatic.  I'm just making a note here.  

So if after your tests of build 71 and these issues are not extant already, I will merge the PR (as discussed with @aron - and since there's no one else to do it), so the fixes goes to the `develop` branch. :smile_cat: 

Can you make a comment on the #359 PR if the issues are resolved already as per your tests?

**Comment by radinamatic** [comment]
@cpauya: the 2 cases I described above now seem to be behaving correctly! :+1: 

One thing that remains is a `Stopped` notification after KA Lite is loaded from the Launchpad:

![ka-stopped](https://cloud.githubusercontent.com/assets/1457929/13586125/83a53e12-e4bf-11e5-829e-52be3665b984.gif)

Could you change that  `Stopped` into `You can start/stop the server from the status icon`?

**Comment by cpauya** [comment]
Hi @radinamatic - Thanks for the update.  I'll work on this after my [release blockers](https://github.com/learningequality/installers/issues?q=is%3Aissue+label%3A%22release+blocker%22+is%3Aopen) tasks.

**Comment by radinamatic** [comment]
:+1: @cpauya, this one is minor annoyance in comparison!

**Comment by cpauya** [comment]
Hi @radina - pls use the [build 89 at Bamboo](http://dungeon.learningequality.org/browse/KL-MT16-89) to test this issue.
